pandas/core/dtypes/concat.py

```diff
@@ -242,6 +242,72 @@ def union_categoricals(to_union, sort_categories=False, ignore_order=False):
         - sort_categories=True and Categoricals are ordered
     ValueError
         Empty list of categoricals passed
+
+    Notes
+    -----
+
+    To learn more about categories, see 'link
+    <http://pandas.pydata.org/pandas-docs/stable/categorical.html#unioning>__`
+
+    Examples
+    --------
+
+    If you want to combine categoricals that do not necessarily have
+    the same categories, `union_categoricals` will combine a list-like
+    of categoricals. The new categories will be the union of the
+    categories being combined.
+
+    >>> a = pd.Categorical(["b", "c"])
+    >>> b = pd.Categorical(["a", "b"])
+    >>> union_categoricals([a, b])
+    [b, c, a, b]
+    Categories (3, object): [b, c, a]
+
+    By default, the resulting categories will be ordered as they appear
+    in the data. If you want the categories to be lexsorted, use
```

Comment:
"resulting categories will be ordered as they appear in the data" is not fully correct (or at least subjective misinterpretable). 
The final categories are not ordered as they appear in the data, but as they appear in the `categories` of the data (ant this is not necessarily the same, as upon construction the categories are sorted, not in order of appearance).

Eg.

```
In [37]: a = pd.Categorical(["c", "b"])

In [38]:  b = pd.Categorical(["a", "b"])

In [39]:  pd.api.types.union_categoricals([a, b])
Out[39]: 
[c, b, a, b]
Categories (3, object): [b, c, a]
```